### DISCUSSION TITLE
DNS and reverse proxy for Wordpress headless

### DISCUSSION THREAD

**Discussion by nikosantis** [vercel/community#3737] [Feature Question]
### Question

I have a website using the app router (Latest version of Next.js) that fetches data from a headless WordPress instance. I have the domain "movilidadcalama.cl" set up on Vercel, which is the website itself. On another hosting provider, I have the domain "samtecno.cl". Within this domain, the subfolder "/pmus" contains the WordPress instance from which I need to retrieve data (samtecno.cl/pmus).

I require assistance with the following:

Configuring the route "admin.movilidadcalama.cl" so that when a user accesses that URL, they are effectively viewing the content at "samtecno.cl/pmus".
Currently, I have a redirection from "admin.movilidadcalama.cl" to "samtecno.cl/pmus/wp-admin".
Additionally, I need the entire navigation on admin.movilidadcalama.cl to mirror the content of samtecno's pmus subdirectory. For instance, "admin.movilidadcalama.cl/wp-admin"...
I've read about the possibility of using a Reverse Proxy or a similar solution. I have access to the other hosting provider where WordPress is installed, so I can edit the .htaccess file or use the DNS/Zone Editor. I also have access to my Vercel Pro account, where my domain is configured.

Could you guide me on how to achieve this? :( 
I have looked for information in several places but I do not understand much.

**Comment by amyegan** [comment]
Hey @nikosantis. You can set up [rewrites in vercel.json](https://vercel.com/docs/projects/project-configuration#rewrites) for the domain and subpaths. It's possible to do something similar using [Middleware](https://vercel.com/docs/functions/edge-middleware) if you you need more flexibility.
